src/models/sunburst.js

```diff
@@ -14,6 +14,7 @@ nv.models.sunburst = function() {
         , id = Math.floor(Math.random() * 10000) //Create semi-unique ID in case user doesn't select one
         , container = null
         , color = nv.utils.defaultColor()
+        , groupColorByParent = true
```

Comment:
Please add info about this new option to the json object at the bottom of examples/documentation.html so that people know what it is and how to use it :)